src/pages/Scheduling/components/CreateScheduleTemplateSheet.tsx

```diff
@@ -431,50 +431,54 @@ export default function CreateScheduleTemplateSheet({
                         )}
                       /> */}
 
-                      <div className="flex items-center gap-0 sm:gap-4 col-span-2 md:col-span-1">
-                        <FormField
-                          control={form.control}
-                          name={`availabilities.${index}.start_time`}
-                          render={({ field }) => (
-                            <FormItem className="flex-1">
-                              <FormLabel required>{t("start_time")}</FormLabel>
-                              <FormControl>
-                                <Input type="time" {...field} />
-                              </FormControl>
-                              <FormMessage />
-                            </FormItem>
-                          )}
-                        />
-
-                        <ArrowRightIcon className="size-4 mt-5" />
+                      <div className="flex items-center gap-0 sm:gap-4 col-span-2 md:col-span-1 h-28">
+                        <div className="flex items-center justify-center mt-16">
```

Comment:
Why add another div with flex wrapping the same contents 🤔  You should be able to address the issue without this.